tests/xfstests/run.pm

```diff
@@ -523,10 +512,6 @@ sub run {
         @tests = shuffle(@tests);
     }
 
-    # Genarate and maintain BLACKLIST by exclude group list
-    my %tests_needto_exclude = exclude_grouplist;
-    my %BLACKLIST = (genarate_blacklist, %tests_needto_exclude);
```

Comment:
We can just keep these codes rather than create `is_in_blacklist`. Otherwise it will call `generate_xfstests_list` at each iteration

Reply:
> We can just keep these codes rather than create `is_in_blacklist`. Otherwise it will call `generate_xfstests_list` at each iteration

Yeah I agree, but the function genarate_blacklist need still move to lib, because if I write it again in generate_result.pm it'll post fail when CI checking. I'll change here to 1 time call to save it in %blacklist first.

Reply:
LGTM

Reply:
Done, I also change the same way with softfail list.